core-tests/src/test/java/org/mule/runtime/core/internal/util/journal/queue/LocalTxQueueTransactionJournalTestCase.java

```diff
@@ -17,50 +16,45 @@
 import static org.hamcrest.Matchers.lessThan;
 import static org.hamcrest.core.Is.is;
 import static org.hamcrest.core.IsNot.not;
+import static org.junit.jupiter.api.Assertions.assertThrows;
```

Comment:
Shouldn't it be `org.junit.Assert.assertThrows`?

Reply:
no, that's the junit 4 one, we should start using the junit 5 ones to ease the eventual transition

Reply:
good to know